ext/oci8/oci8_interface.c

```diff
@@ -152,18 +128,10 @@ PHP_FUNCTION(oci_define_by_name)
 	/* if (zend_hash_add(statement->defines, name, name_len, define, sizeof(php_oci_define), (void **)&tmp_define) == SUCCESS) { */
 	zvtmp = zend_string_init(name, name_len, 0);
 	if ((define = zend_hash_add_new_ptr(statement->defines, zvtmp, define)) != NULL) {
-#if PHP_VERSION_ID < 70300
 		zend_string_release(zvtmp);
```

Comment:
Was keeping the non *_ex() calls deliberate?

Reply:
Actually no, let me change those back